hack/e2e.go

```diff
@@ -207,6 +207,11 @@ func run(deploy deployer) error {
 	}
 
 	if *up {
+		// If we tried to bring the Kubemark cluster up, make a courtesy
+		// attempt to bring it down so we're not leaving resources around.
+		if *down {
+			defer xmlWrap("TearDown", deploy.Down)
```

Comment:
Yes. The same logic (with teardown being called twice) we had before. This is "as expected".
https://github.com/fejta/kubernetes/blob/ffe611569dbd645a7d5c9428828ece1404e7e396/hack/jenkins/e2e-runner.sh#L169-L181